packages/editor-ui/src/views/TestDefinition/TestDefinitionRootView.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+import { useWorkflowHelpers } from '@/composables/useWorkflowHelpers';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { useRouter } from 'vue-router';
+import { onMounted } from 'vue';
+const router = useRouter();
+const workflowHelpers = useWorkflowHelpers({ router });
+const workflowStore = useWorkflowsStore();
+
+async function initWorkflow() {
+	const workflowId = router.currentRoute.value.params.name as string;
+	const isAlreadyInitialized = workflowStore.workflow.id === workflowId;
```

Comment:
Could it be already in the process of fetching workflow at this point? `isAlreadyInitialized` will still be false in that case and the second request will be triggered.